apps-rendering/src/liveBlock.ts

```diff
@@ -16,25 +19,42 @@ type LiveBlock = {
 	firstPublished: Option<Date>;
 	lastModified: Option<Date>;
 	body: Body;
+	contributors: Contributor[];
 };
 
 // ----- Functions ----- //
 
+const contributorTags = (contributors: string[], tags: Tag[]): Tag[] => {
+	const isTag = (tag: Tag | undefined): tag is Tag => tag !== undefined;
+	return contributors
+		.map((contributor) =>
+			tags.find((tag) => tag.id === `profile/${contributor}`),
+		)
+		.filter(isTag);
+};
```

Comment:
FYI the `isTag` function could be written in a generic way to make it useful in other circumstances:

```ts
const isDefined = <A>(a: A | undefined): a is A => a !== undefined;
```

I think replacing `isTag` with `isDefined` gives you the same result, but `isDefined` could be moved to `lib.ts` and prove useful in similar scenarios.

Not necessary for this PR, but might make a nice optimisation?

Reply:
Great suggestion, I'll put it in the backlog so as not to block this PR. 
I'll also card up the 2 follow-up points from the PR description.